tachyon/zk/plonk/permutation/permutation_keys.h

```diff
@@ -0,0 +1,117 @@
+#ifndef TACHYON_ZK_PLONK_PERMUTATION_KEYS_H_
+#define TACHYON_ZK_PLONK_PERMUTATION_KEYS_H_
+
+#include "tachyon/base/buffer/copyable.h"
+#include "tachyon/export.h"
+#include "tachyon/math/elliptic_curves/affine_point.h"
+#include "tachyon/math/polynomials/univariate/univariate_polynomial.h"
+
+namespace tachyon {
+namespace zk {
+
+template <typename Curve>
+class PermutationVerifyingKey;
+
+template <typename Curve>
+class PermutationVerifyingKey {
+ public:
+  using CommitmentTy = math::AffinePoint<Curve>;
+  using CommitmentVec = std::vector<CommitmentTy>;
```

Comment:
```suggestion
  using Commitment = math::AffinePoint<Curve>;
  using Commitments = std::vector<CommitmentTy>;
```

Reply:
Or I think you can just use the type directly without alias since the alias type is not that shorter.

Reply:
`AffinePoint` used an alias to indicate that it is a `Commitment`. And I will change the name of the alias to something like suggestion.